modules/katamari/src/test/ts/atomic/api/arr/ArrFindMapTest.ts

```diff
@@ -1,54 +1,56 @@
-import { Assert, UnitTest } from '@ephox/bedrock-client';
+import { describe, it } from '@ephox/bedrock-client';
 import fc from 'fast-check';
 import * as Arr from 'ephox/katamari/api/Arr';
 import * as Fun from 'ephox/katamari/api/Fun';
 import { Optional } from 'ephox/katamari/api/Optional';
-import { tOptional } from 'ephox/katamari/api/OptionalInstances';
 import * as Optionals from 'ephox/katamari/api/Optionals';
+import { assertNone, assertSome } from 'ephox/katamari/test/AssertOptional';
 
-UnitTest.test('Arr.findMap of empty is none', () => {
-  Assert.eq('eq', Optional.none(), Arr.findMap([], Fun.die('⊥')), tOptional());
-});
+describe('atomic.katamari.api.arr.ArrFindMapTest', () => {
+  it('Arr.findMap of empty is none', () => {
+    assertNone(Arr.findMap([], Fun.die('⊥')));
+  });
 
-UnitTest.test('Arr.findMap of non-empty is first if f is Optional.some', () => {
-  fc.assert(fc.property(
-    fc.integer(),
-    fc.array(fc.integer()),
-    (head, tail) => {
-      const arr = [ head, ...tail ];
-      Assert.eq('eq', Optional.some(head), Arr.findMap(arr, Optional.some), tOptional());
-    }
-  ));
-});
+  it('Arr.findMap of non-empty is first if f is Optional.some', () => {
+    fc.assert(fc.property(
+      fc.integer(),
+      fc.array(fc.integer()),
+      (head, tail) => {
+        const arr = [ head, ...tail ];
+        assertSome(Arr.findMap(arr, Optional.some), head);
+      }
+    ));
+  });
 
-UnitTest.test('Arr.findMap of non-empty is none if f is Optional.none', () => {
-  fc.assert(fc.property(
-    fc.array(fc.integer()),
-    (arr) => {
-      Assert.eq('eq', Optional.none(), Arr.findMap(arr, () => Optional.none()), tOptional());
-    }
-  ));
-});
+  it('Arr.findMap of non-empty is none if f is Optional.none', () => {
+    fc.assert(fc.property(
+      fc.array(fc.integer()),
+      (arr) => {
+        assertNone(Arr.findMap(arr, () => Optional.none()));
```

Comment:
ETA reduce? Turns out it was like this beforehand, so ignore if you want
```suggestion
        assertNone(Arr.findMap(arr, Optional.none));
```